iOS_4x_release.md

```diff
@@ -0,0 +1,42 @@
+# Aligning major version releases for the Adobe iOS SDK
+
+### Why are we getting another major update already?
+
+Apple has started doing an annual requirement to increase the minimum version of Xcode used when submitting to the App Store.  Each Xcode version only supports back to a specific minimum version of iOS.  Loosly speaking, Xcode version and _minimum_ iOS version are correlated.  
+
+https://developer.apple.com/ios/submit/
+
+In order to align with Apple's changing requirements, Adobe will also be doing an annual major release of its iOS SDKs.  In this yearly major update, you can expect the following changes:
+  - Increasing the minimum supported version of iOS.
+  - Increasing the minimum supported version of Xcode.
+  - Possible increase to minimum supported version of Swift.
+
+### Why are all the extensions updating to major version 4?
+
+The main goal is to simplify the Adobe SDK distibution.  Aliging all extesions on the same major version allows us to create predictability.  For example:
+  - All Adobe-owned SDK extensions that share a major version are compatible with one another.
+  - Each major version of an Adobe-owned SDK extension has a designated minimum OS version.
+
+### Why did some of the extensions skip major versions?
+
+You may be wondering "what happened to versions 2 and 3 of AEPEdge?"  Our apologies for any confusion this may cause.  In order to achieve our goal of unifying major versions, some extensions, AEPEdge for example, had to skip one or more major versions.  Hopefully the our explaination above has shed light on this anomoly.
```

Comment:
Fix grammar and typos - `Hopefully our explanation above has shed light on this anomaly.`